src/web/js/log.js

```diff
@@ -22,9 +22,12 @@ var IndexedDBBackend = function() {
 
 var logger = (function(backend) {
   function guid() {
-    var array = new Uint32Array(4);
+    var array = new Uint32Array(6);
     window.crypto.getRandomValues(array);
-    return array;
+    return array.reduce(
```

Comment:
This `reduce` method isn't defined in Safari on `Uint32Array`, which makes the whole page fail to load.